http-clients/netty-nio-client/src/test/java/software/amazon/awssdk/http/nio/netty/NettyNioAsyncHttpClientWireMockTest.java

```diff
@@ -126,7 +128,15 @@ public void defaultConnectionIdleTimeout() {
     }
 
     @Test
-    public void noTlsTimeout_shouldResolveToConnectTimeout() {
+    public void defaultTlsHandshakeTimeout() {
+        try (NettyNioAsyncHttpClient client = (NettyNioAsyncHttpClient) NettyNioAsyncHttpClient.create()) {
+            assertThat(client.configuration().tlsHandshakeTimeout()).
+                isEqualTo(GLOBAL_HTTP_DEFAULTS.get(TLS_NEGOTIATION_TIMEOUT));
```

Comment:
Nit: Consider hard-coding the expected value in the test for regression detection. Otherwise a bug introduced to `GLOBAL_HTTP_DEFAULTS` would go undetected.